React/src/App.js

```diff
@@ -44,8 +39,8 @@ function App() {
         </Item>
         <Item title="Role" icon="isnotblank" badge="new">
           <RadioGroup 
-            items={radioGroupItems}
-            value="Owner"
+            items={employeeData.roles}
+            value={employeeData.roles[0]}
```

Comment:
I suspect that `value` should be `defaultValue` here and in the TextArea above. Otherwise, we would have to handle `onValueChange`.